### ISSUE TITLE
npm:@google-cloud/vision

### ISSUE THREAD

**Issue by smunipalli** [denoland/deno#20183]
google cloud vision API is failing with the following error

Not implemented: Http2Session.ref
Uncaught TypeError: error sending request for url (http://vision.googleapis.com:443/google.cloud.vision.v1.....

**Comment by bartlomieju** [comment]
This is related to #17987 and missing `node:http2` support that I'm working on in #19671

**Comment by satyarohith** [comment]
I tried running it with the latest deno and it doesn't work yet. We're getting a bad request response back from GCP and the response isn't propagated correctly to the client. Should be fixed by #23755 

```
➜  google_sdk GRPC_VERBOSITY=debug GRPC_TRACE=all denod run -A --unstable-byonm index.js
D 2024-05-12T10:33:10.251Z | v1.10.7 77922 | resolving_load_balancer | dns:vision.googleapis.com:443 IDLE -> IDLE
D 2024-05-12T10:33:10.258Z | v1.10.7 77922 | connectivity_state | (1) dns:vision.googleapis.com:443 IDLE -> IDLE
D 2024-05-12T10:33:10.258Z | v1.10.7 77922 | dns_resolver | Resolver constructed for target dns:vision.googleapis.com:443
D 2024-05-12T10:33:10.259Z | v1.10.7 77922 | channel | (1) dns:vision.googleapis.com:443 Channel constructed with options {
  "grpc.max_receive_message_length": -1,
  "grpc.max_send_message_length": -1,
  "grpc.initial_reconnect_backoff_ms": 1000
}
D 2024-05-12T10:33:10.260Z | v1.10.7 77922 | channel_stacktrace | (1) Channel constructed 
    at new InternalChannel (file:///Users/sr/p/google_sdk/node_modules/@grpc/grpc-js/build/src/internal-channel.js:248:23)
    at new ChannelImplementation (file:///Users/sr/p/google_sdk/node_modules/@grpc/grpc-js/build/src/channel.js:35:32)
    at new Client (file:///Users/sr/p/google_sdk/node_modules/@grpc/grpc-js/build/src/client.js:66:36)
    at new ServiceClientImpl (file:///Users/sr/p/google_sdk/node_modules/@grpc/grpc-js/build/src/make-client.js:58:5)
    at GrpcClient.createStub (file:///Users/sr/p/google_sdk/node_modules/google-gax/build/src/grpc.js:344:22)
    at eventLoopTick (ext:core/01_core.js:207:9)
D 2024-05-12T10:33:10.260Z | v1.10.7 77922 | resolving_load_balancer | dns:vision.googleapis.com:443 IDLE -> IDLE
D 2024-05-12T10:33:10.260Z | v1.10.7 77922 | connectivity_state | (2) dns:vision.googleapis.com:443 IDLE -> IDLE
D 2024-05-12T10:33:10.260Z | v1.10.7 77922 | dns_resolver | Resolver constructed for target dns:vision.googleapis.com:443
D 2024-05-12T10:33:10.261Z | v1.10.7 77922 | channel | (2) dns:vision.googleapis.com:443 Channel constructed with options {
  "grpc.max_receive_message_length": -1,
  "grpc.max_send_message_length": -1,
  "grpc.initial_reconnect_backoff_ms": 1000
}
D 2024-05-12T10:33:10.261Z | v1.10.7 77922 | channel_stacktrace | (2) Channel constructed 
    at new InternalChannel (file:///Users/sr/p/google_sdk/node_modules/@grpc/grpc-js/build/src/internal-channel.js:248:23)
    at new ChannelImplementation (file:///Users/sr/p/google_sdk/node_modules/@grpc/grpc-js/build/src/channel.js:35:32)
    at new Client (file:///Users/sr/p/google_sdk/node_modules/@grpc/grpc-js/build/src/client.js:66:36)
    at new ServiceClientImpl (file:///Users/sr/p/google_sdk/node_modules/@grpc/grpc-js/build/src/make-client.js:58:5)
    at GrpcClient.createStub (file:///Users/sr/p/google_sdk/node_modules/google-gax/build/src/grpc.js:344:22)
    at eventLoopTick (ext:core/01_core.js:207:9)
D 2024-05-12T10:33:10.262Z | v1.10.7 77922 | channel | (2) dns:vision.googleapis.com:443 createResolvingCall [0] method="/google.cloud.vision.v1.ImageAnnotator/BatchAnnotateImages", deadline=2024-05-12T10:43:10.261Z
D 2024-05-12T10:33:10.262Z | v1.10.7 77922 | resolving_call | [0] Created
D 2024-05-12T10:33:10.262Z | v1.10.7 77922 | resolving_call | [0] Deadline: 2024-05-12T10:43:10.261Z
D 2024-05-12T10:33:10.262Z | v1.10.7 77922 | resolving_call | [0] Deadline will be reached in 599999ms
D 2024-05-12T10:33:10.262Z | v1.10.7 77922 | resolving_call | [0] start called
D 2024-05-12T10:33:10.262Z | v1.10.7 77922 | dns_resolver | Looking up DNS hostname vision.googleapis.com
D 2024-05-12T10:33:10.294Z | v1.10.7 77922 | resolving_load_balancer | dns:vision.googleapis.com:443 IDLE -> CONNECTING
D 2024-05-12T10:33:10.294Z | v1.10.7 77922 | connectivity_state | (2) dns:vision.googleapis.com:443 IDLE -> CONNECTING
D 2024-05-12T10:33:10.294Z | v1.10.7 77922 | channel | (2) dns:vision.googleapis.com:443 callRefTimer.ref | configSelectionQueue.length=1 pickQueue.length=0
D 2024-05-12T10:33:10.294Z | v1.10.7 77922 | resolving_call | [0] startRead called
D 2024-05-12T10:33:10.297Z | v1.10.7 77922 | resolving_call | [0] write() called with message of length 15347
D 2024-05-12T10:33:10.297Z | v1.10.7 77922 | resolving_call | [0] halfClose called
D 2024-05-12T10:33:10.317Z | v1.10.7 77922 | dns_resolver | Resolved addresses for target dns:vision.googleapis.com:443: [172.217.163.202:443,142.250.196.170:443,142.250.196.74:443,142.250.196.42:443,142.250.196.10:443,142.250.195.234:443,142.250.195.202:443,142.250.195.170:443,142.250.195.138:443,142.250.195.106:443,142.250.195.74:443,142.250.195.42:443,142.250.71.42:443,216.58.196.170:443,172.217.160.138:443,172.217.31.202:443,2404:6800:4007:82a::200a:443,2404:6800:4007:82b::200a:443,2404:6800:4007:82c::200a:443,2404:6800:4007:811::200a:443]
D 2024-05-12T10:33:10.318Z | v1.10.7 77922 | subchannel | (1) 172.217.163.202:443 Subchannel constructed with options {
  "grpc.max_receive_message_length": -1,
  "grpc.max_send_message_length": -1,
  "grpc.initial_reconnect_backoff_ms": 1000
}
D 2024-05-12T10:33:10.318Z | v1.10.7 77922 | subchannel_refcount | (1) 172.217.163.202:443 refcount 0 -> 1
D 2024-05-12T10:33:10.318Z | v1.10.7 77922 | subchannel | (2) [2404:6800:4007:82a::200a]:443 Subchannel constructed with options {
  "grpc.max_receive_message_length": -1,
  "grpc.max_send_message_length": -1,
  "grpc.initial_reconnect_backoff_ms": 1000
}
D 2024-05-12T10:33:10.318Z | v1.10.7 77922 | subchannel_refcount | (2) [2404:6800:4007:82a::200a]:443 refcount 0 -> 1
D 2024-05-12T10:33:10.318Z | v1.10.7 77922 | subchannel | (3) 142.250.196.170:443 Subchannel constructed with options {
  "grpc.max_receive_message_length": -1,
  "grpc.max_send_message_length": -1,
  "grpc.initial_reconnect_backoff_ms": 1000
}
D 2024-05-12T10:33:10.318Z | v1.10.7 77922 | subchannel_refcount | (3) 142.250.196.170:443 refcount 0 -> 1
D 2024-05-12T10:33:10.318Z | v1.10.7 77922 | subchannel | (4) [2404:6800:4007:82b::200a]:443 Subchannel constructed with options {
  "grpc.max_receive_message_length": -1,
  "grpc.max_send_message_length": -1,
  "grpc.initial_reconnect_backoff_ms": 1000
}
D 2024-05-12T10:33:10.318Z | v1.10.7 77922 | subchannel_refcount | (4) [2404:6800:4007:82b::200a]:443 refcount 0 -> 1
D 2024-05-12T10:33:10.319Z | v1.10.7 77922 | subchannel | (5) 142.250.196.74:443 Subchannel constructed with options {
  "grpc.max_receive_message_length": -1,
  "grpc.max_send_message_length": -1,
  "grpc.initial_reconnect_backoff_ms": 1000
}
D 2024-05-12T10:33:10.319Z | v1.10.7 77922 | subchannel_refcount | (5) 142.250.196.74:443 refcount 0 -> 1
D 2024-05-12T10:33:10.319Z | v1.10.7 77922 | subchannel | (6) [2404:6800:4007:82c::200a]:443 Subchannel constructed with options {
  "grpc.max_receive_message_length": -1,
  "grpc.max_send_message_length": -1,
  "grpc.initial_reconnect_backoff_ms": 1000
}
D 2024-05-12T10:33:10.319Z | v1.10.7 77922 | subchannel_refcount | (6) [2404:6800:4007:82c::200a]:443 refcount 0 -> 1
D 2024-05-12T10:33:10.319Z | v1.10.7 77922 | subchannel | (7) 142.250.196.42:443 Subchannel constructed with options {
  "grpc.max_receive_message_length": -1,
  "grpc.max_send_message_length": -1,
  "grpc.initial_reconnect_backoff_ms": 1000
}
D 2024-05-12T10:33:10.319Z | v1.10.7 77922 | subchannel_refcount | (7) 142.250.196.42:443 refcount 0 -> 1
D 2024-05-12T10:33:10.319Z | v1.10.7 77922 | subchannel | (8) [2404:6800:4007:811::200a]:443 Subchannel constructed with options {
  "grpc.max_receive_message_length": -1,
  "grpc.max_send_message_length": -1,
  "grpc.initial_reconnect_backoff_ms": 1000
}
D 2024-05-12T10:33:10.319Z | v1.10.7 77922 | subchannel_refcount | (8) [2404:6800:4007:811::200a]:443 refcount 0 -> 1
D 2024-05-12T10:33:10.319Z | v1.10.7 77922 | subchannel | (9) 142.250.196.10:443 Subchannel constructed with options {
  "grpc.max_receive_message_length": -1,
  "grpc.max_send_message_length": -1,
  "grpc.initial_reconnect_backoff_ms": 1000
}
D 2024-05-12T10:33:10.319Z | v1.10.7 77922 | subchannel_refcount | (9) 142.250.196.10:443 refcount 0 -> 1
D 2024-05-12T10:33:10.319Z | v1.10.7 77922 | subchannel | (10) 142.250.195.234:443 Subchannel constructed with options {
  "grpc.max_receive_message_length": -1,
  "grpc.max_send_message_length": -1,
  "grpc.initial_reconnect_backoff_ms": 1000
}
D 2024-05-12T10:33:10.319Z | v1.10.7 77922 | subchannel_refcount | (10) 142.250.195.234:443 refcount 0 -> 1
D 2024-05-12T10:33:10.319Z | v1.10.7 77922 | subchannel | (11) 142.250.195.202:443 Subchannel constructed with options {
  "grpc.max_receive_message_length": -1,
  "grpc.max_send_message_length": -1,
  "grpc.initial_reconnect_backoff_ms": 1000
}
D 2024-05-12T10:33:10.319Z | v1.10.7 77922 | subchannel_refcount | (11) 142.250.195.202:443 refcount 0 -> 1
D 2024-05-12T10:33:10.319Z | v1.10.7 77922 | subchannel | (12) 142.250.195.170:443 Subchannel constructed with options {
  "grpc.max_receive_message_length": -1,
  "grpc.max_send_message_length": -1,
  "grpc.initial_reconnect_backoff_ms": 1000
}
D 2024-05-12T10:33:10.319Z | v1.10.7 77922 | subchannel_refcount | (12) 142.250.195.170:443 refcount 0 -> 1
D 2024-05-12T10:33:10.319Z | v1.10.7 77922 | subchannel | (13) 142.250.195.138:443 Subchannel constructed with options {
  "grpc.max_receive_message_length": -1,
  "grpc.max_send_message_length": -1,
  "grpc.initial_reconnect_backoff_ms": 1000
}
D 2024-05-12T10:33:10.319Z | v1.10.7 77922 | subchannel_refcount | (13) 142.250.195.138:443 refcount 0 -> 1
D 2024-05-12T10:33:10.319Z | v1.10.7 77922 | subchannel | (14) 142.250.195.106:443 Subchannel constructed with options {
  "grpc.max_receive_message_length": -1,
  "grpc.max_send_message_length": -1,
  "grpc.initial_reconnect_backoff_ms": 1000
}
D 2024-05-12T10:33:10.319Z | v1.10.7 77922 | subchannel_refcount | (14) 142.250.195.106:443 refcount 0 -> 1
D 2024-05-12T10:33:10.319Z | v1.10.7 77922 | subchannel | (15) 142.250.195.74:443 Subchannel constructed with options {
  "grpc.max_receive_message_length": -1,
  "grpc.max_send_message_length": -1,
  "grpc.initial_reconnect_backoff_ms": 1000
}
D 2024-05-12T10:33:10.319Z | v1.10.7 77922 | subchannel_refcount | (15) 142.250.195.74:443 refcount 0 -> 1
D 2024-05-12T10:33:10.319Z | v1.10.7 77922 | subchannel | (16) 142.250.195.42:443 Subchannel constructed with options {
  "grpc.max_receive_message_length": -1,
  "grpc.max_send_message_length": -1,
  "grpc.initial_reconnect_backoff_ms": 1000
}
D 2024-05-12T10:33:10.319Z | v1.10.7 77922 | subchannel_refcount | (16) 142.250.195.42:443 refcount 0 -> 1
D 2024-05-12T10:33:10.319Z | v1.10.7 77922 | subchannel | (17) 142.250.71.42:443 Subchannel constructed with options {
  "grpc.max_receive_message_length": -1,
  "grpc.max_send_message_length": -1,
  "grpc.initial_reconnect_backoff_ms": 1000
}
D 2024-05-12T10:33:10.319Z | v1.10.7 77922 | subchannel_refcount | (17) 142.250.71.42:443 refcount 0 -> 1
D 2024-05-12T10:33:10.319Z | v1.10.7 77922 | subchannel | (18) 216.58.196.170:443 Subchannel constructed with options {
  "grpc.max_receive_message_length": -1,
  "grpc.max_send_message_length": -1,
  "grpc.initial_reconnect_backoff_ms": 1000
}
D 2024-05-12T10:33:10.319Z | v1.10.7 77922 | subchannel_refcount | (18) 216.58.196.170:443 refcount 0 -> 1
D 2024-05-12T10:33:10.319Z | v1.10.7 77922 | subchannel | (19) 172.217.160.138:443 Subchannel constructed with options {
  "grpc.max_receive_message_length": -1,
  "grpc.max_send_message_length": -1,
  "grpc.initial_reconnect_backoff_ms": 1000
}
D 2024-05-12T10:33:10.319Z | v1.10.7 77922 | subchannel_refcount | (19) 172.217.160.138:443 refcount 0 -> 1
D 2024-05-12T10:33:10.319Z | v1.10.7 77922 | subchannel | (20) 172.217.31.202:443 Subchannel constructed with options {
  "grpc.max_receive_message_length": -1,
  "grpc.max_send_message_length": -1,
  "grpc.initial_reconnect_backoff_ms": 1000
}
D 2024-05-12T10:33:10.319Z | v1.10.7 77922 | subchannel_refcount | (20) 172.217.31.202:443 refcount 0 -> 1
D 2024-05-12T10:33:10.319Z | v1.10.7 77922 | subchannel_refcount | (1) 172.217.163.202:443 refcount 1 -> 2
D 2024-05-12T10:33:10.319Z | v1.10.7 77922 | subchannel_refcount | (2) [2404:6800:4007:82a::200a]:443 refcount 1 -> 2
D 2024-05-12T10:33:10.319Z | v1.10.7 77922 | subchannel_refcount | (3) 142.250.196.170:443 refcount 1 -> 2
D 2024-05-12T10:33:10.319Z | v1.10.7 77922 | subchannel_refcount | (4) [2404:6800:4007:82b::200a]:443 refcount 1 -> 2
D 2024-05-12T10:33:10.319Z | v1.10.7 77922 | subchannel_refcount | (5) 142.250.196.74:443 refcount 1 -> 2
D 2024-05-12T10:33:10.319Z | v1.10.7 77922 | subchannel_refcount | (6) [2404:6800:4007:82c::200a]:443 refcount 1 -> 2
D 2024-05-12T10:33:10.319Z | v1.10.7 77922 | subchannel_refcount | (7) 142.250.196.42:443 refcount 1 -> 2
D 2024-05-12T10:33:10.319Z | v1.10.7 77922 | subchannel_refcount | (8) [2404:6800:4007:811::200a]:443 refcount 1 -> 2
D 2024-05-12T10:33:10.319Z | v1.10.7 77922 | subchannel_refcount | (9) 142.250.196.10:443 refcount 1 -> 2
D 2024-05-12T10:33:10.319Z | v1.10.7 77922 | subchannel_refcount | (10) 142.250.195.234:443 refcount 1 -> 2
D 2024-05-12T10:33:10.319Z | v1.10.7 77922 | subchannel_refcount | (11) 142.250.195.202:443 refcount 1 -> 2
D 2024-05-12T10:33:10.319Z | v1.10.7 77922 | subchannel_refcount | (12) 142.250.195.170:443 refcount 1 -> 2
D 2024-05-12T10:33:10.319Z | v1.10.7 77922 | subchannel_refcount | (13) 142.250.195.138:443 refcount 1 -> 2
D 2024-05-12T10:33:10.319Z | v1.10.7 77922 | subchannel_refcount | (14) 142.250.195.106:443 refcount 1 -> 2
D 2024-05-12T10:33:10.320Z | v1.10.7 77922 | subchannel_refcount | (15) 142.250.195.74:443 refcount 1 -> 2
D 2024-05-12T10:33:10.320Z | v1.10.7 77922 | subchannel_refcount | (16) 142.250.195.42:443 refcount 1 -> 2
D 2024-05-12T10:33:10.320Z | v1.10.7 77922 | subchannel_refcount | (17) 142.250.71.42:443 refcount 1 -> 2
D 2024-05-12T10:33:10.320Z | v1.10.7 77922 | subchannel_refcount | (18) 216.58.196.170:443 refcount 1 -> 2
D 2024-05-12T10:33:10.320Z | v1.10.7 77922 | subchannel_refcount | (19) 172.217.160.138:443 refcount 1 -> 2
D 2024-05-12T10:33:10.320Z | v1.10.7 77922 | subchannel_refcount | (20) 172.217.31.202:443 refcount 1 -> 2
D 2024-05-12T10:33:10.320Z | v1.10.7 77922 | pick_first | Start connecting to subchannel with address 172.217.163.202:443
D 2024-05-12T10:33:10.320Z | v1.10.7 77922 | pick_first | IDLE -> CONNECTING
D 2024-05-12T10:33:10.320Z | v1.10.7 77922 | resolving_load_balancer | dns:vision.googleapis.com:443 CONNECTING -> CONNECTING
D 2024-05-12T10:33:10.320Z | v1.10.7 77922 | connectivity_state | (2) dns:vision.googleapis.com:443 CONNECTING -> CONNECTING
D 2024-05-12T10:33:10.320Z | v1.10.7 77922 | channel | (2) dns:vision.googleapis.com:443 callRefTimer.unref | configSelectionQueue.length=0 pickQueue.length=0
D 2024-05-12T10:33:10.320Z | v1.10.7 77922 | subchannel | (1) 172.217.163.202:443 IDLE -> CONNECTING
D 2024-05-12T10:33:10.321Z | v1.10.7 77922 | transport | dns:vision.googleapis.com:443 creating HTTP/2 session to 172.217.163.202:443
D 2024-05-12T10:33:10.337Z | v1.10.7 77922 | channel | (2) dns:vision.googleapis.com:443 createRetryingCall [1] method="/google.cloud.vision.v1.ImageAnnotator/BatchAnnotateImages"
D 2024-05-12T10:33:10.337Z | v1.10.7 77922 | resolving_call | [0] Created child [1]
D 2024-05-12T10:33:10.337Z | v1.10.7 77922 | retrying_call | [1] start called
D 2024-05-12T10:33:10.337Z | v1.10.7 77922 | channel | (2) dns:vision.googleapis.com:443 createLoadBalancingCall [2] method="/google.cloud.vision.v1.ImageAnnotator/BatchAnnotateImages"
D 2024-05-12T10:33:10.337Z | v1.10.7 77922 | retrying_call | [1] Created child call [2] for attempt 1
D 2024-05-12T10:33:10.337Z | v1.10.7 77922 | load_balancing_call | [2] start called
D 2024-05-12T10:33:10.337Z | v1.10.7 77922 | load_balancing_call | [2] Pick called
D 2024-05-12T10:33:10.337Z | v1.10.7 77922 | load_balancing_call | [2] Pick result: QUEUE subchannel: null status: undefined undefined
D 2024-05-12T10:33:10.337Z | v1.10.7 77922 | channel | (2) dns:vision.googleapis.com:443 callRefTimer.ref | configSelectionQueue.length=0 pickQueue.length=1
D 2024-05-12T10:33:10.337Z | v1.10.7 77922 | retrying_call | [1] startRead called
D 2024-05-12T10:33:10.337Z | v1.10.7 77922 | load_balancing_call | [2] startRead called
D 2024-05-12T10:33:10.338Z | v1.10.7 77922 | retrying_call | [1] write() called with message of length 15352
D 2024-05-12T10:33:10.338Z | v1.10.7 77922 | load_balancing_call | [2] write() called with message of length 15352
D 2024-05-12T10:33:10.338Z | v1.10.7 77922 | retrying_call | [1] halfClose called
D 2024-05-12T10:33:10.399Z | v1.10.7 77922 | subchannel | (1) 172.217.163.202:443 CONNECTING -> READY
D 2024-05-12T10:33:10.399Z | v1.10.7 77922 | pick_first | Pick subchannel with address 172.217.163.202:443
D 2024-05-12T10:33:10.399Z | v1.10.7 77922 | subchannel_refcount | (1) 172.217.163.202:443 refcount 2 -> 3
D 2024-05-12T10:33:10.399Z | v1.10.7 77922 | subchannel_refcount | (1) 172.217.163.202:443 refcount 3 -> 2
D 2024-05-12T10:33:10.399Z | v1.10.7 77922 | subchannel_refcount | (2) [2404:6800:4007:82a::200a]:443 refcount 2 -> 1
D 2024-05-12T10:33:10.399Z | v1.10.7 77922 | subchannel_refcount | (3) 142.250.196.170:443 refcount 2 -> 1
D 2024-05-12T10:33:10.399Z | v1.10.7 77922 | subchannel_refcount | (4) [2404:6800:4007:82b::200a]:443 refcount 2 -> 1
D 2024-05-12T10:33:10.399Z | v1.10.7 77922 | subchannel_refcount | (5) 142.250.196.74:443 refcount 2 -> 1
D 2024-05-12T10:33:10.399Z | v1.10.7 77922 | subchannel_refcount | (6) [2404:6800:4007:82c::200a]:443 refcount 2 -> 1
D 2024-05-12T10:33:10.399Z | v1.10.7 77922 | subchannel_refcount | (7) 142.250.196.42:443 refcount 2 -> 1
D 2024-05-12T10:33:10.399Z | v1.10.7 77922 | subchannel_refcount | (8) [2404:6800:4007:811::200a]:443 refcount 2 -> 1
D 2024-05-12T10:33:10.399Z | v1.10.7 77922 | subchannel_refcount | (9) 142.250.196.10:443 refcount 2 -> 1
D 2024-05-12T10:33:10.399Z | v1.10.7 77922 | subchannel_refcount | (10) 142.250.195.234:443 refcount 2 -> 1
D 2024-05-12T10:33:10.399Z | v1.10.7 77922 | subchannel_refcount | (11) 142.250.195.202:443 refcount 2 -> 1
D 2024-05-12T10:33:10.399Z | v1.10.7 77922 | subchannel_refcount | (12) 142.250.195.170:443 refcount 2 -> 1
D 2024-05-12T10:33:10.399Z | v1.10.7 77922 | subchannel_refcount | (13) 142.250.195.138:443 refcount 2 -> 1
D 2024-05-12T10:33:10.399Z | v1.10.7 77922 | subchannel_refcount | (14) 142.250.195.106:443 refcount 2 -> 1
D 2024-05-12T10:33:10.399Z | v1.10.7 77922 | subchannel_refcount | (15) 142.250.195.74:443 refcount 2 -> 1
D 2024-05-12T10:33:10.399Z | v1.10.7 77922 | subchannel_refcount | (16) 142.250.195.42:443 refcount 2 -> 1
D 2024-05-12T10:33:10.399Z | v1.10.7 77922 | subchannel_refcount | (17) 142.250.71.42:443 refcount 2 -> 1
D 2024-05-12T10:33:10.399Z | v1.10.7 77922 | subchannel_refcount | (18) 216.58.196.170:443 refcount 2 -> 1
D 2024-05-12T10:33:10.399Z | v1.10.7 77922 | subchannel_refcount | (19) 172.217.160.138:443 refcount 2 -> 1
D 2024-05-12T10:33:10.399Z | v1.10.7 77922 | subchannel_refcount | (20) 172.217.31.202:443 refcount 2 -> 1
D 2024-05-12T10:33:10.399Z | v1.10.7 77922 | pick_first | CONNECTING -> READY
D 2024-05-12T10:33:10.399Z | v1.10.7 77922 | resolving_load_balancer | dns:vision.googleapis.com:443 CONNECTING -> READY
D 2024-05-12T10:33:10.399Z | v1.10.7 77922 | channel | (2) dns:vision.googleapis.com:443 callRefTimer.unref | configSelectionQueue.length=0 pickQueue.length=0
D 2024-05-12T10:33:10.400Z | v1.10.7 77922 | load_balancing_call | [2] Pick called
D 2024-05-12T10:33:10.400Z | v1.10.7 77922 | load_balancing_call | [2] Pick result: COMPLETE subchannel: (1) 172.217.163.202:443 status: undefined undefined
D 2024-05-12T10:33:10.400Z | v1.10.7 77922 | connectivity_state | (2) dns:vision.googleapis.com:443 CONNECTING -> READY
D 2024-05-12T10:33:10.650Z | v1.10.7 77922 | transport_flowctrl | (1) 172.217.163.202:443 local window size: undefined remote window size: undefined
Warning: Not implemented: Http2Session.socket
D 2024-05-12T10:33:10.651Z | v1.10.7 77922 | transport_internals | (1) 172.217.163.202:443 session.closed=false session.destroyed=false session.socket.destroyed=undefined
D 2024-05-12T10:33:10.651Z | v1.10.7 77922 | load_balancing_call | [2] Created child call [3]
D 2024-05-12T10:33:10.651Z | v1.10.7 77922 | subchannel_call | [3] write() called with message of length 15352
D 2024-05-12T10:33:10.651Z | v1.10.7 77922 | subchannel_call | [3] sending data chunk of length 15352
D 2024-05-12T10:33:10.654Z | v1.10.7 77922 | load_balancing_call | [2] halfClose called
D 2024-05-12T10:33:10.654Z | v1.10.7 77922 | subchannel_call | [3] end() called
D 2024-05-12T10:33:10.654Z | v1.10.7 77922 | subchannel_call | [3] calling end() on HTTP/2 stream
D 2024-05-12T10:33:10.755Z | v1.10.7 77922 | subchannel_call | [3] Received server headers:
                :status: 200
                content-type: application/grpc
                grpc-status: 13
                grpc-message: 400:Bad Request
                content-length: 0
                date: Sun, 12 May 2024 10:33:09 GMT
                alt-svc: h3=":443"; ma=2592000,h3-29=":443"; ma=2592000

D 2024-05-12T10:33:10.755Z | v1.10.7 77922 | load_balancing_call | [2] Received metadata
D 2024-05-12T10:33:10.755Z | v1.10.7 77922 | retrying_call | [1] Received metadata from child [2]
D 2024-05-12T10:33:10.755Z | v1.10.7 77922 | retrying_call | [1] Committing call [2] at index 0
D 2024-05-12T10:33:10.755Z | v1.10.7 77922 | resolving_call | [0] Received metadata
D 2024-05-12T10:33:20.319Z | v1.10.7 77922 | subchannel_refcount | (2) [2404:6800:4007:82a::200a]:443 refcount 1 -> 0
D 2024-05-12T10:33:20.320Z | v1.10.7 77922 | subchannel_refcount | (3) 142.250.196.170:443 refcount 1 -> 0
D 2024-05-12T10:33:20.320Z | v1.10.7 77922 | subchannel_refcount | (4) [2404:6800:4007:82b::200a]:443 refcount 1 -> 0
D 2024-05-12T10:33:20.320Z | v1.10.7 77922 | subchannel_refcount | (5) 142.250.196.74:443 refcount 1 -> 0
D 2024-05-12T10:33:20.320Z | v1.10.7 77922 | subchannel_refcount | (6) [2404:6800:4007:82c::200a]:443 refcount 1 -> 0
D 2024-05-12T10:33:20.320Z | v1.10.7 77922 | subchannel_refcount | (7) 142.250.196.42:443 refcount 1 -> 0
D 2024-05-12T10:33:20.320Z | v1.10.7 77922 | subchannel_refcount | (8) [2404:6800:4007:811::200a]:443 refcount 1 -> 0
D 2024-05-12T10:33:20.320Z | v1.10.7 77922 | subchannel_refcount | (9) 142.250.196.10:443 refcount 1 -> 0
D 2024-05-12T10:33:20.320Z | v1.10.7 77922 | subchannel_refcount | (10) 142.250.195.234:443 refcount 1 -> 0
D 2024-05-12T10:33:20.320Z | v1.10.7 77922 | subchannel_refcount | (11) 142.250.195.202:443 refcount 1 -> 0
D 2024-05-12T10:33:20.320Z | v1.10.7 77922 | subchannel_refcount | (12) 142.250.195.170:443 refcount 1 -> 0
D 2024-05-12T10:33:20.320Z | v1.10.7 77922 | subchannel_refcount | (13) 142.250.195.138:443 refcount 1 -> 0
D 2024-05-12T10:33:20.320Z | v1.10.7 77922 | subchannel_refcount | (14) 142.250.195.106:443 refcount 1 -> 0
D 2024-05-12T10:33:20.320Z | v1.10.7 77922 | subchannel_refcount | (15) 142.250.195.74:443 refcount 1 -> 0
D 2024-05-12T10:33:20.320Z | v1.10.7 77922 | subchannel_refcount | (16) 142.250.195.42:443 refcount 1 -> 0
D 2024-05-12T10:33:20.320Z | v1.10.7 77922 | subchannel_refcount | (17) 142.250.71.42:443 refcount 1 -> 0
D 2024-05-12T10:33:20.320Z | v1.10.7 77922 | subchannel_refcount | (18) 216.58.196.170:443 refcount 1 -> 0
D 2024-05-12T10:33:20.320Z | v1.10.7 77922 | subchannel_refcount | (19) 172.217.160.138:443 refcount 1 -> 0
D 2024-05-12T10:33:20.320Z | v1.10.7 77922 | subchannel_refcount | (20) 172.217.31.202:443 refcount 1 -> 0
```

**Comment by satyarohith** [comment]
This is fixed by #24016
